utils/saasdial/dial.go

```diff
@@ -19,46 +19,60 @@ package saasdial
 
 import (
 	"context"
-	"net"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
 	"time"
 
 	"github.com/percona/pmm/utils/tlsconfig"
-	"github.com/percona/pmm/version"
-	"github.com/pkg/errors"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials"
 )
 
 const dialTimeout = 10 * time.Second
 
-// Dial creates gRPC connection to Percona Platform
-func Dial(ctx context.Context, sessionID string, hostPort string) (*grpc.ClientConn, error) {
-	host, _, err := net.SplitHostPort(hostPort)
+// Dial creates http/https POST request to Percona Platform
+func Dial(ctx context.Context, endpoint, accessToken string, body io.Reader) ([]byte, error) {
```

Comment:
i think it should be renamed to "MakeRequest" or "MakePlatformRequest" or `DoRequest` since we are now making HTTP calls using it

i think we should add another argument for HTTP method, because some orgd endpoints are GET, PUT, DELETE so in case this method will be used in the future to call those i think it would be better to have a method arg

Reply:
Func renamed (also package) and added another param to pass http method.